lib/skate/detours/db/detour.ex

```diff
@@ -162,6 +162,7 @@ defmodule Skate.Detours.Db.Detour do
         :author_id,
         :activated_at,
         :updated_at,
+        :status,
```

Comment:
suggestion: can we remove `:state_value` with the addition of this?

Reply:
I'd like to see if we can do this but otherwise approved!

Reply:
It was used to make the `categorize_detour` call cheaper https://github.com/mbta/skate/blob/bf-query-with-status-field/lib/skate/detours/detours.ex#L123-L125

Reply:
Let's chat about this? https://github.com/mbta/skate/pull/2965

Reply:
Good for now in this PR though